corehq/apps/userreports/tests/test_report_filters.py

```diff
@@ -215,24 +215,26 @@ def test_string_datatype(self):
         self.filter_spec['datatype'] = 'string'
         filter = ReportFilterFactory.from_spec(self.filter_spec)
         tests = (
-            (1, '1'),
-            (1.2, '1.2'),
+            ('1', '1'),
+            ('1.2', '1.2'),
```

Comment:
don't think I had anything else in mind - I think I was more probably unit testing the underlying conversion through the wrong API (should have just tested the function, not through the filter).